.github/workflows/nightly-cd.yml

```diff
@@ -101,6 +101,10 @@ jobs:
       - name: Ensure all package versions are consistent
         run: rush ensure-consistent-versions
 
+      # Alpha version run latest
+      - name: Force build flavor to `beta`
```

Comment:
Is this to mitigate somebody checking in the .env file set differently?

Reply:
It's poorly named, it sets all the deps in the package.json. So where we have `@azure/communication-calling: 1.x.x-beta.1 || 1.y.y` it rewrites and outputs only `@azure/communication-calling: 1.x.x-beta.1`